pyo3-ffi/src/lib.rs

```diff
@@ -255,6 +189,8 @@
 #![doc = concat!("[manual_builds]: https://pyo3.rs/v", env!("CARGO_PKG_VERSION"), "/building-and-distribution.html#manual-builds \"Manual builds - Building and Distribution - PyO3 user guide\"")]
 //! [setuptools-rust]: https://github.com/PyO3/setuptools-rust "Setuptools plugin for Rust extensions"
 //! [PEP 384]: https://www.python.org/dev/peps/pep-0384 "PEP 384 -- Defining a Stable ABI"
+//! [`sequential`]: https://github.com/PyO3/pyo3/tree/main/examples/sequential "An example illustrating multi-phase initialization"
+//! [`examples`]: https://github.com/PyO3/pyo3/tree/main/examples/sequential "PyO3 examples"
```

Comment:
I think this might work to pin to the exact version:

```suggestion
#![doc = concat!("[`sequential`]: https://github.com/PyO3/pyo3/tree/v", env!("CARGO_PKG_VERSION"), "/examples/sequential \"An example illustrating multi-phase initialization\"")]
#![doc = concat!("[`examples`]: https://github.com/PyO3/pyo3/tree/v", env!("CARGO_PKG_VERSION"), "/examples/sequential \"PyO3 examples\"")]
```